vhdbuilder/cgmanifest.json

```diff
@@ -10,6 +10,17 @@
                 }
             },
             "DevelopmentDependency": false
+        },
+        {
+            "Component": {
+                "Type": "other",
+                "Other": {
+                    "Name": "ContainerDWindows",
```

Comment:
In this repo, Linux is also using containerd, but we here just cover windows contained only